src/asm2wasm.h

```diff
@@ -342,6 +342,7 @@ class Asm2WasmBuilder {
     
   TrapMode trapMode;
   PassOptions passOptions;
+  bool runJsFfiPass;
```

Comment:
That's fine with me. Do you want me to use a similar name for the emscripten and fastcomp options? "legalize" seemed a bit specific to binaryen so that's why I moved towards a more generic naming.